sunbeam-python/sunbeam/commands/openstack.py

```diff
@@ -334,6 +334,18 @@ def run(self, status: Optional[Status] = None) -> Result:
                 ),
             }
         )
+
+        # NOTE(jamespage)
+        # When dedicated control nodes are used, ceph is not enabled during
+        # bootstrap - however storage nodes may be added later so re-assess
+        # as to whether ceph integrations can be enabled during resize.
+        storage_nodes = client.cluster.list_nodes_by_role("storage")
+        if storage_nodes:
+            tf_vars["enable-ceph"] = True
+            tf_vars["ceph-offer-url"] = f"{CONTROLLER_MODEL}.{MICROCEPH_APPLICATION}"
```

Comment:
ceph-replication-osd-count can  also be set here instead of in https://github.com/canonical/snap-openstack/blob/main/sunbeam-python/sunbeam/commands/openstack.py#L332